critiquebrainz/ws/review/views.py

```diff
@@ -231,21 +233,30 @@ def review_modify_handler(review_id, user):
 
     :resheader Content-Type: *application/json*
     """
-
+    # TODO(psolanki): One caveat is that client will have to pass the unmodified parameter with previous revision's value
+    #                 otherwise it will be set to None
```

Comment:
Need some help to with this. I tried some solutions but they were not promising.